src/main/scala/io/reactivesocket/tck/DSLTestClient.scala

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2016 Facebook, Inc.
+ * <p>
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ *  the License. You may obtain a copy of the License at
+ *  <p>
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *  <p>
+ *  Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ *  an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ *  specific language governing permissions and limitations under the License.
+ */
+
+package io.reactivesocket.tck
+
+import java.io.PrintWriter
+
+object ClientIDGen {
+  var count = 0
+
+  def currentCount(): Int = {
+    count += 1
```

Comment:
This confuses me, the method currentCount() doesn't sound like it should change the count.

Can you add some documentation around what this means.